README.md

```diff
@@ -6,69 +6,63 @@ vSphere Integrated Containers Engine (VIC Engine) is a container runtime for vSp
 
 See [VIC Engine Architecture](doc/design/arch/arch.md) for a high level overview.
 
-
 ## Project Status
 
+[commands]:https://vmware.github.io/vic-product/assets/files/html/1.3/vic_app_dev/container_operations.html
+[ctrust]:https://docs.docker.com/engine/security/trust/content_trust
+[wizard]:https://vmware.github.io/vic-product/assets/files/html/1.3/vic_vsphere_admin/deploy_vch_client.html
+[dch]:https://vmware.github.io/vic-product/assets/files/html/1.3/vic_vsphere_admin/deploy_vch_dchphoton.html
+
 VIC Engine now provides:
-* basic function for most of the core lifecycle operations: pull, create, start, attach, run, kill, stop, wait, rm, logs.
-* early vCenter support, leveraging DRS for initial placement.
-* volume support [--volume-store](doc/user/usage.md#configuring-volumes-in-a-virtual-container-host) - SIOC is not integrated but can be set as normal.
+* support for most of the Docker commands for core container, image, volume and network lifecycle operations. Several `docker compose` commands are also supported. See the complete list of supported commands [here][commands].
+* vCenter support, leveraging DRS for initial placement. vMotion is also supported.
+* volume support for vSphere datastores (including vSAN) and NFS shares. See [--volume-store](doc/user/usage.md#configuring-volumes-in-a-virtual-container-host) - SIOC is not integrated but can be set as normal.
```

Comment:
Ping @matthewavery for reviewing this part.

Reply:
is the [here][commands] mean to be a hyperlink? if so It might be missing the link :)

Reply:
`vsphere datastores` may not be quite correct. we do support standard vmfs backed datastores. We do support vsan datastores, NFS datastores, and ISCSI datastores. However, things like VVOL datastores and any other kind of specialty datastores are not supported. :)

Reply:
The `[commands]` hyperlink is defined here: https://github.com/vmware/vic/pull/6974/files#diff-04c6e90faac2675aa89e2176d2eec7d8R11